README.md

```diff
@@ -1,143 +1,23 @@
-# QMENTA SDK TOOL TUTORIAL
+# QMENTA SDK Tool Tutorial
 
 <img src="assets/qmenta_logo.png" alt="QMENTA" style="width: 50%">
 
 ---
 
-## Introduction
+This repository contains the data, code, scripts and configuration files associated with the [Getting Started tutorial](https://docs.qmenta.com/sdk/getting_started.html) in the [QMENTA SDK documentation](https://docs.qmenta.com/sdk/) portal.
 
-In this repository we will guide you through the process of deploying a tool into the  [QMENTA platform](https://client.qmenta.com/#/login) and then run it.
+## Description
 
-To get more detailed and technical information about the QMENTA SDK and its capabilities, please refer to it's [documentation](https://docs.qmenta.com/sdk/).
+The tutorial helps you learn the foundations of using the QMENTA SDK and the technological ecosystem around it,
+including Docker and the [QMENTA platform](https://client.qmenta.com/#/login).
 
-As a case study, we will implement a tool that takes as inputs an oncology medical image and a segmentation mask with one or more labels, and then it uses the [Pyradiomics library](https://pyradiomics.readthedocs.io/en/latest/index.html) to extract radiomic features from the data. 
-The tool will allow the user to select which classes of radiomic features wants to compute and also select.
+As a case study, the tutorial goes over the implementation of a tool that takes as inputs an oncology medical image and a segmentation mask with one or more labels, and then uses the [pyradiomics library] to extract radiomic features from the data.
+The tool will allow the user to select which classes of radiomic features they want to compute and also select a
+variety of image filters to be applied before extracting the radiomic features.
 
-Pyradiomics is an open-source python package for the extraction of Radiomics features from medical imaging. To learn more about the aim of the project and the features of the package take a look to the following publication: [van Griethuysenet al. 2017](https://doi.org/10.1158/0008-5472.CAN-17-0339)
+[pyradiomics library]: https://pyradiomics.readthedocs.io/en/latest/
 
+Pyradiomics is an open-source Python package for the extraction of radiomic features from medical images.
+To learn more about the aim of the project and the features of the package you can read the following [publication](https://doi.org/10.1158/0008-5472.CAN-17-0339).
 
-
-The deployment of a tool has three principal steps:
```

Comment:
Why is this section deleted? To add a tool, it is still needed to write the script, create a docker image, and add the tool to platform.

Reply:
The reason why I have removed most of the README.md information, including these lines, is that the revamp of the QMENTA SDK docs includes now a Getting Started guide with that information and much more.
Here you have a sneak peek of what it will look like:

![image](https://user-images.githubusercontent.com/14277223/100248152-6dfff900-2f3b-11eb-8cb6-754b2962bf59.png)

Reply:
ok, that's fine then. Was that done in a different PR? I looked in this one but didn't see the information added somewhere else.

Reply:
It will be included in the PR I will create for the `qmenta-sdk` repository in our internal BitBucket. I'll let you know when that is done, first I wanted to close this PR because it is smaller and I need to have those files in the `master` branch in order to finish up the documentation.

Reply:
Right. I will approve this one then.